File: Cpp/FlowControl/if.cpp
#include <iostream>

using namespace std;

int main()
{
    int number;
    cout << "Enter an integer: ";
    cin >> number;

    // true if number is less than 0
    if (number < 0)
    {
        cout << "You entered " << number << " and it is less than zero." << endl;
    }
    // true if number is greater than 0
    if (number > 0)
    {
        cout << "You entered " << number << " and it is greater than zero." << endl;
    }
    cout << "The if statement is easy." << endl;

    return 0;
}

File: Python/Flowcontrol/if.py
def main():
    number = int(input("Enter an integer: "))

    # true if number is less than 0
    if number < 0:
        print(f"You entered {number} and it is less than zero.")
    
    # true if number is greater than 0
    if number > 0:
        print(f"You entered {number} and it is greater than zero.")
    
    print("The if statement is easy.")

if __name__ == "__main__":
    main()